src/main/java/com/hazelcast/cloud/Client.java

```diff
@@ -118,26 +113,24 @@ private static void sqlExample(HazelcastInstance client) {
         }
         System.out.println("--------------------");
 
-        System.out.println("Retrieving a city name via SQL...");
+        System.out.println("Retrieving the capital name via SQL...");
         try (SqlResult result = client.getSql()
-            .execute("SELECT __key, this FROM cities WHERE __key = ?", "United States")) {
+                .execute("SELECT __key, this FROM capitals WHERE __key = ?", "United States")) {
 
             for (SqlRow row : result) {
                 String country = row.getObject("__key");
                 String city = row.getObject("this");
-                System.out.printf("Country name: %s; City name: %s\n", country, city);
+                System.out.printf("Country name: %s; Capital name: %s\n", country, city);
             }
         }
         System.out.println("--------------------");
-        System.exit(0);
     }
 
     /**
-     * This example shows how to work with Hazelcast SQL queries via Maps that contains
-     * JSON serialized values.
-     * - Select single json element data from a Map
-     * - Select data from Map with filtering
-     * - Join data from 2 Maps and select json elements
+     * This example shows how to work with Hazelcast SQL queries via Maps that
+     * contains JSON serialized values. - Select single json element data from a
+     * Map - Select data from Map with filtering - Join data from 2 Maps and
```

Comment:
I guess there's a small formatting issue here. The sentences start with a dash.

Reply:
corrected the Javadoc formatting with proper lists